packages/ui/src/stories/Button.stories.tsx

```diff
@@ -0,0 +1,66 @@
+// Adapted from https://github.com/shadcn-ui/ui/blob/94ee191d989cf93246f2feaca68b1fdb24c18940/apps/www/registry/stories/button.stories.tsx
+
+import type { Meta, StoryObj } from "@storybook/react";
+
+import React from "react";
+import { fn } from "@storybook/test";
+
+import { Button } from "../button";
+import { Loader2, Mail } from "../icon";
+
+const meta: Meta<typeof Button> = {
+	title: "Button",
+	component: Button,
+	tags: ["autodocs"],
+	argTypes: {},
+	args: { children: "Button", onClick: fn() },
+};
+export default meta;
+
+type Story = StoryObj<typeof Button>;
+
+export const Base: Story = {
+	args: { children: "Button" },
+};
+export const Outline: Story = {
+	args: {
+		variant: "outline",
+	},
+};
+export const Ghost: Story = {
+	args: {
+		variant: "ghost",
+	},
+};
+export const Secondary: Story = {
+	args: {
+		variant: "secondary",
+	},
+};
+export const Link: Story = {
+	args: {
+		variant: "link",
+	},
+};
+export const Loading: Story = {
+	render: (args) => (
+		<Button {...args}>
+			{/* animate-spin not working? */}
+			<Loader2 className="mr-2 h-4 w-4 animate-spin" />
```

Comment:
it's a mystery to me why the `animate-spin` class isn't working here... the other tailwind classes seem to work ok

Reply:
mystery solved: `animate-spin` was only being used by the storybook file, which isn't included in the tailwind config. having tailwind also monitor storybook files makes things work

Reply:
Should we have a separate config for storybook maybe? Not sure if that's something that's done, but don't the story book classes then end up in the prod css, even if they aren't used maybe?

Reply:
yeah I think you're right—I haven't found a way to add that tailwind stuff for only storybook yet, but I'll keep looking!

Reply:
Maybe setting up a storybook app instead of colocating the storybook instance in `ui` like how `turbo` suggests below could work?
Then we would indeed have a separate `tailwind.config` for storybook, which would not affect the `ui`.
See here?: https://github.com/gmickel/turborepo-shadcn-nextjs/blob/main/apps/storybook/tailwind.config.ts